clients/hadoopfs/src/main/java/io/lakefs/storage/SimpleStorageAccessStrategy.java

```diff
@@ -37,7 +37,13 @@ public SimpleStorageAccessStrategy(LakeFSFileSystem lakeFSFileSystem, LakeFSClie
 
     @Override
     public FSDataOutputStream createDataOutputStream(ObjectLocation objectLocation,
-            CreateOutputStreamParams params) throws ApiException, IOException {
+                                                     CreateOutputStreamParams params) throws ApiException, IOException {
+        return createDataOutputStream(objectLocation, params, true);
+    }
```

Comment:
This should now become an [interface default method](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html).  I think those are already available in our Java versions.